h2o-algos/src/main/java/hex/mojo/GenericModelMojoWriter.java

```diff
@@ -0,0 +1,42 @@
+package hex.mojo;
+
+import hex.ModelMojoWriter;
+import hex.genmodel.utils.IOUtils;
+import water.fvec.ByteVec;
+import water.util.Log;
+import java.io.*;
+
+public class GenericModelMojoWriter extends ModelMojoWriter<GenericModel, GenericModelParameters, GenericModelOutput> {
+
+    private final ByteVec _mojoBytes;
+
+    public GenericModelMojoWriter(ByteVec _mojoBytes) {
+        this._mojoBytes = _mojoBytes;
+    }
+
+
+    @Override
+    public String mojoVersion() {
+        return "1.0";
+    }
+
+    @Override
+    protected void writeModelData() throws IOException {
+        // Do nothing on purpose
+    }
+
+    @Override
+    public void writeTo(OutputStream os) {
+        try(final InputStream inputStream = _mojoBytes.openStream(null)) {
+            IOUtils.copyStream(inputStream,os);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Do nothing?

Reply:
No, definitely do not do nothing ! Resolved in https://github.com/h2oai/h2o-3/pull/3274/commits/f1e98871e6f824392d3364e668ed61884d5dd783.